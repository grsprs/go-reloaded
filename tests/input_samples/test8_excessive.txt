only two words here (up, 5) total