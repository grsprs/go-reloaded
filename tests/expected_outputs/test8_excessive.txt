ONLY TWO words here total